css/index.css

```diff
@@ -143,4 +266,79 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+a:hover {
+    color:blue;
+}
+
+/* CSS for product specific page */
+
+.middle-container {
+	display:flex;
+	flex-wrap:wrap;
+	justify-content: space-between;
+    padding-top:30px;
+    border-top: 2px dashed green;
+}
+
+.product-mid-content {
+	border: 3px double purple;
+	margin-bottom:35px;
+	width:48%;
+	background-color:honeydew;
+	padding:20px;
+}
+
+.product-mid-content button {
+	border:1px solid black;
+	padding: 6px 40px;
+	margin-top:20px;
+	background-color:white;
+    border-radius:10%;
+    cursor: pointer;
+}
+
+.product-mid-content button:hover {
+    background-color:black;
+    color:white;
+}
+
+.product-mid-content button:active {
+    background-color:black;
+    color:white;
+    box-shadow: 0 6px orange;
+    transform: transformY(4px);
+}
+
+/* contact page specific CSS below */
```

Comment:
I see you put all CSS on the same page. A great habit to get into early would be to create css pages for each additional page you create. For example, a `contact.css` page.